pkg/apis/kops/ntpconfig.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kops
+
+// NTPConfig is the configuration for NTP.
+type NTPConfig struct {
+	// SkipInstall is set to true when we prevent kops from installing and modifying NTP.
+	SkipInstall bool `json:"skipInstall,omitempty"`
```

Comment:
I think we were planning to move away from `SkipInstall` and add `Enabled` and/or `Managed`:
`Enabled` = false would mean the service is actually disabled by the task.
`Managed` = false would just skip the task.

/cc @justinsb @rifelpet @olemarkus

Reply:
Thanks for the review, @hakman ! I've added `Managed`. I wasn't sure we want to have `Enabled` for NTP since I believe either `systemd-timesyncd` or `chrony` is needed in every env. Does this make sense?

Reply:
👍